packaging/ansible-runner-service-project/project/roles/ovirt-ova-pack/tasks/main.yml

```diff
@@ -1,18 +1,19 @@
 ---
 - block:
   - name: Create temporary directory
-    tempfile:
+    ansible.builtin.tempfile:
       state: directory
       suffix: ova
     register: ova_temp
 
   - name: Copy pack_ova.py to temp directory
-    copy:
+    ansible.builtin.copy:
       src: pack_ova.py
       dest: "{{ ova_temp.path }}/pack_ova.py"
+      mode: preserve
```

Comment:
@mz-pdm Shouldn't we rather make it more clear by using mode: 755?

Reply:
Yes.

Reply:
fixed- https://github.com/oVirt/ovirt-engine/pull/684/commits/32e58e9d6c352ca05b24a377a47e740248c0db6e